recipes-kernel/accton-as4630-54pe-cpld-mod/accton-as4630-54pe-cpld-mod.bb

```diff
@@ -0,0 +1,15 @@
+SUMMARY = "This module supports cpld that read and write"
```

Comment:
Maybe use a better summary, e.g. the first sentence in the comment block in the c file.